packages/@aws-cdk/aws-glue/package.json

```diff
@@ -156,8 +156,7 @@
       "docs-public-apis:@aws-cdk/aws-glue.ClassificationString.PARQUET",
       "docs-public-apis:@aws-cdk/aws-glue.ClassificationString.ORC",
       "docs-public-apis:@aws-cdk/aws-glue.ClassificationString.value",
-      "docs-public-apis:@aws-cdk/aws-glue.JobExecutable.bind",
-      "no-unused-type:@aws-cdk/aws-glue.JobState"
+      "events-method-signature:@aws-cdk/aws-glue.Job.onStateChange"
```

Comment:
what error is this suppressing?

Reply:
onXxx() methods are expected to have a certain signature (id, options). In this case that method has (id, jobState, options)